### PR TITLE
Postulate that snippet tags be closed

### PR DESCRIPTION
**Pull request by carmenbianca**
Related to https://github.com/fsfe/reuse-tool/issues/1074

### CODE REVIEW

File: site/content/en/spec-3.3.md [hunk 1]
```diff
@@ -177,12 +177,14 @@ An example of a comment header:
 #### In-line Snippet comments
 
 Sometimes Licensing Information only applies to a certain Snippet instead of the
-whole Commentable File. For these cases SPDX snippet tags MUST be used for that
+whole Commentable File. For these cases, SPDX snippet tags MUST be used for that
 Snippet (as defined in [SPDX Specification, Annex
```
Comment (carmenbianca) [inline site/content/en/spec-3.3.md:181]: Eh, even when corrected, this language is awkward. Let's reword.

Comment (carmenbianca) [reply site/content/en/spec-3.3.md:181]: Disambiguate: make SPDX snippets optional

**Review by silverhook — APPROVED** [review]
LGTM. Maybe dobule-check if  “Snippets” vs “snippets” (case) is coherently used in the section.

**Comment by carmenbianca** [comment]
Fixed, thank you!

**Pull request merged**